libs/python.json

```diff
@@ -199,5 +202,6 @@
   {"id":"py.pendulum","imports":["pendulum"],"name":"pendulum","repo":"sdispater/pendulum","tags":["datetime"],"tech":["utils"],"status":"awaiting-model"},
   {"id":"py.psutil","imports":["psutil"],"name":"psutil","repo":"giampaolo/psutil","tags":["system"],"tech":["utils"],"status":"awaiting-model"},
   {"id":"py.python-dateutil","imports":["dateutil"],"name":"dateutil","repo":"dateutil/dateutil","tags":["date-time"],"tech":["utils"],"status":"awaiting-model"},
-  {"id":"py.python-slugify","imports":["slugify"],"name":"Slugify","repo":"un33k/python-slugify","tags":["unicode-slugs"],"tech":["utils"],"status":"awaiting-model"}
+  {"id":"py.python-slugify","imports":["slugify"],"name":"Slugify","repo":"un33k/python-slugify","tags":["unicode-slugs"],"tech":["utils"],"status":"awaiting-model"},
+  {"id":"py.supervisor","imports":["xmlrpclib"],"name":"supervisor","repo":"Supervisor/supervisor","tags":["process-control"],"tech":["utils"],"status":"awaiting-model"}
```

Comment:
so these are plugins for supervisor? are they very popular?

Reply:
not sure whether this is a plugin or the system itself :) but they expose python API http://supervisord.org/api.html. It has to be next popular to already supported libraries according to libraries.io

Reply:
most peculiar